examples/google-signin/google-signin-iframe.html

```diff
@@ -0,0 +1,25 @@
+<body>
+  <!--
+    Specify a Google Sign-In OAuth Client ID,
+    then load Google Sign-In's script.
+    https://developers.google.com/identity/sign-in/web/sign-in
+  -->
```

Comment:
I would also add a comment as close as you can to the `content` that gives the developers a big note about what they should be replacing: `NOTE: Replace content with your own Google Sign-In OAuth Client ID.`

Reply:
Updated!